lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -514,5 +516,21 @@ Ufe::Selection recreateDescendants(const Ufe::Selection& src, const Ufe::Path& f
     return dst;
 }
 
+Ufe::Rtid getUsdUfeRuntimeId()
+{
+    if (usdUfeRtid == 0) {
+        try {
+            usdUfeRtid = UFE_NS::RunTimeMgr::instance().getId(USD_UFE_RUNTIME_NAME);
+        }
+        // This shoudl catch ufe's InvalidRunTimeName exception, but they don't
```

Comment:
No longer relevant, but a bit confused by this: InvalidRunTimeName is exposed in ufeExcept.h.